apps/aecore/lib/aecore/structures/voting_tx.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Aecore.Structures.VotingTx do
+  alias Aecore.Structures.VotingTx
+
+  @type t :: %VotingTx{
+    data: VotingQuestionTx.t() | VotingAnswerTx.t()
```

Comment:
maybe choose a different name than `data` as it is already used in the parent object, how about `voting_payload`

Reply:
`VotingQuestionTx.t() | VotingAnswerTx.t()` could be extracted as  `@type voting_payload_types :: VotingQuestionTx.t() | VotingAnswerTx.t()`